### ISSUE TITLE
Accessibility Report: Elements

### ISSUE THREAD

**Issue by sherakama** [SU-SWS/decanter#312]
Use the AMP tool to scan the [Helpers](http://decanter.stanford.edu/section-elements.html#kssref-elements) section for accessibility issues and document them to this ticket.

**Comment by sherakama** [comment]
AMP Report: https://stanford.levelaccess.net/public/reporting/view_report.php?report_id=283430

No violations found.

**Comment by sherakama** [comment]
Chrome report: 94

Missing title in iframe.

![Screen Shot 2019-03-13 at 9 14 14 PM](https://user-images.githubusercontent.com/550602/54330944-fc501200-45d4-11e9-853f-cb3a55ca7d48.png)
